### DISCUSSION TITLE
<Error code 145>

### DISCUSSION THREAD

**Discussion by yasintawasulli** [community/community#45987] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,
I am trying to clean and install my maven to create image but after running a while I get this error code in my intelliJ console

Builder lifecycle 'creator' failed with status code 145

can you help me finding my error,

my application has micro services architecture

**Comment by ozcanyldzhn** [comment]
Hi,

The error "Builder lifecycle 'creator' failed with status code 145" in IntelliJ usually occurs when building a containerized application using Maven and Buildpacks. Here are a few things you can check to resolve the issue:

Check Disk Space:

Status code 145 can sometimes indicate that there is not enough disk space. Run:


df -h
to check available space. If necessary, free up space and try again.
Ensure Docker is Running:

If your application is containerized, make sure Docker is running properly before building.
Increase Memory Allocation for Docker:

If your application has multiple microservices, it may require more memory. Increase Docker's allocated memory from Docker Desktop settings.
Clear Maven Cache and Rebuild:

Try cleaning your Maven cache and rebuilding the project:


mvn clean package
Run the Build Command with Debug Mode:

This will give you more detailed error logs:


mvn -X spring-boot:build-image
Check the logs for more specific errors.
If the issue persists, please share your full error log to better diagnose the problem.

Hope this helps!
